### ISSUE TITLE
Modifier key states are inverted in WASM demo

### ISSUE THREAD

**Issue by YgorSouza** [emilk/egui#2715]
**Describe the bug**

In the raw input display of the web egui demo app, the states of the modifier keys are inverted. When a key is first pressed, its state remains `false` and then changes to `true` when it is released. It remains in this inverted mode until a different key is pressed, at which point the modifier display its correct state until it changes again. This behavior is not observed when running the demo app locally on Linux.

**To Reproduce**
1. Go to https://www.egui.rs/#demo
2. Open the Backend tab and select the Inspection window
3. Open the Input > Raw input collapsible
4. Press the modifier keys (Ctrl, Alt, Shift, Cmd) with and without other keys and observe the displayed states

**Expected behavior**

The Modifiers UI should display `true` when a key is pressed and `false` when it is not, as it does in the desktop demo.

**Screenshots**

**Desktop (please complete the following information):**
 - OS:  Fedora Linux 37
 - Browser  Firefox
 - Version  109.0.1

**Additional context**

As mentioned, I could not reproduce the issue locally by building for desktop. I only see it in the web demo, which I am not able to build myself. But I don't see any commits addressing that since the last deployment of the web demo in 1384fa3287afdbf0c9993b015d03fa56258d8156, so I am opening the issue so someone more experienced with wasm can take a look at it.

**Comment by emilk** [comment]
I can't reproduce in my browser.

You should be able to build the web demo by running [`sh/build_demo_web.sh`](https://github.com/emilk/egui/blob/master/sh/build_demo_web.sh). You can serve it with [`sh/start_server.sh`](https://github.com/emilk/egui/blob/master/sh/start_server.sh).

**Comment by YgorSouza** [comment]
So I built the wasm app from master and I have the same behavior as the online demo. But I also tested on a Windows machine (also with Firefox 109.0.1, as well as Edge) and it's working normally. It also works fine on a Debian 11 virtual machine with Firefox 102.7.0esr. However, I tried on a Fedora virtual machine (on a different computer) and got the same issue. Then I tried on Chromium 109 on the same Fedora VM and it worked normally. So apparently this a problem with Firefox 109 specifically on Linux. It's probably not egui's fault, so I think this issue can be closed.
